Tests are broken

```
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
        panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x4851d4]

```

@aeperez24 

```
?   	aeperez24/videoScrapper	[no test files]
?   	aeperez24/videoScrapper/application	[no test files]
?   	aeperez24/videoScrapper/mock/animeshow	[no test files]
?   	aeperez24/videoScrapper/mock/port	[no test files]
?   	aeperez24/videoScrapper/mock/service	[no test files]
?   	aeperez24/videoScrapper/port	[no test files]
2022/08/30 08:07:01 downloading fromlocation
--- FAIL: TestDownloadFromLink (0.00s)
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x4851d4]

goroutine 9 [running]:
testing.tRunner.func1.2({0x6170a0, 0x801510})
	/home/rodrigo/go/go1.18.3/src/testing/testing.go:1389 +0x24e
testing.tRunner.func1()
	/home/rodrigo/go/go1.18.3/src/testing/testing.go:1392 +0x39f
panic({0x6170a0, 0x801510})
	/home/rodrigo/go/go1.18.3/src/runtime/panic.go:838 +0x207
io.ReadAll({0x0, 0x0})
	/home/rodrigo/go/go1.18.3/src/io/io.go:645 +0xd4
io/ioutil.ReadAll(...)
	/home/rodrigo/go/go1.18.3/src/io/ioutil/ioutil.go:27
aeperez24/videoScrapper/provider/animeshow.TestDownloadFromLink(0x0?)
	/home/rodrigo/dev/github/videoScrapper/provider/animeshow/downloader_service_test.go:68 +0x97
testing.tRunner(0xc0001e4340, 0x670ca8)
	/home/rodrigo/go/go1.18.3/src/testing/testing.go:1439 +0x102
created by testing.(*T).Run
	/home/rodrigo/go/go1.18.3/src/testing/testing.go:1486 +0x35f
FAIL	aeperez24/videoScrapper/provider/animeshow	0.009s
ok  	aeperez24/videoScrapper/provider/cuevana	(cached)
ok  	aeperez24/videoScrapper/service	(cached)
FAIL
```

--- a/provider/animeshow/downloader_service_test.go
+++ b/provider/animeshow/downloader_service_test.go
@@ -26,7 +26,7 @@ const MIRROR_LINK = "https://www2.animeshow.tv/Tate-no-Yuusha-no-Nariagari-Seaso
 func TestGetEpisodesAvaliable(t *testing.T) {
 	ds := DowloaderService{
 		ScrapService:     ScrapperServiceImpl{},
-		HttpWrapper:      buildHttpWrapperMock(),
+		HttpWrapper:      buildHttpWrapperMock(t),
 		AppConfiguration: service.AppConfiguration{},
 	}
 	episodesList, _ := ds.getEpisodesAvaliable(SERIE_LINK)
@@ -38,7 +38,7 @@ func TestGetEpisodesAvaliable(t *testing.T) {
 func TestGetSortedEpisodesAvaliable(t *testing.T) {
 	ds := DowloaderService{
 		ScrapService:     ScrapperServiceImpl{},
-		HttpWrapper:      buildHttpWrapperMock(),
+		HttpWrapper:      buildHttpWrapperMock(t),
 		AppConfiguration: service.AppConfiguration{},
 	}
 	episodesList, _ := ds.GetSortedEpisodesAvaliable(SERIE_LINK)
@@ -49,19 +49,21 @@ func TestGetSortedEpisodesAvaliable(t *testing.T) {
 func TestDownloadFromM4upload(t *testing.T) {
 	ds := DowloaderService{
 		ScrapService:     ScrapperServiceImpl{},
-		HttpWrapper:      buildHttpWrapperMock(),
+		HttpWrapper:      buildHttpWrapperMock(t),
 		AppConfiguration: service.AppConfiguration{},
 	}
 
-	reader, _, _ := ds.downloadFromM4upload(DOWNLOAD_URL)
-	result, _ := ioutil.ReadAll(reader)
+	reader, _, err := ds.downloadFromM4upload(DOWNLOAD_URL)
+	assert.NoError(t, err)
+	result, err := ioutil.ReadAll(reader)
+	assert.NoError(t, err)
 	assert.Equal(t, VIDEO_DOWNLOAD, string(result))
 }
 
 func TestDownloadFromLink(t *testing.T) {
 	ds := DowloaderService{
 		ScrapService:     ScrapperServiceImpl{},
-		HttpWrapper:      buildHttpWrapperMock(),
+		HttpWrapper:      buildHttpWrapperMock(t),
 		AppConfiguration: service.AppConfiguration{},
 	}
 	reader, _, _ := ds.DownloadEpisodeFromLink(SERIE_LINK, "1")
@@ -69,18 +71,18 @@ func TestDownloadFromLink(t *testing.T) {
 	assert.Equal(t, VIDEO_DOWNLOAD, string(result))
 }
 
-func buildHttpWrapperMock() service.HttpWrapper {
+func buildHttpWrapperMock(t *testing.T) service.HttpWrapper {
 	httpWrapper := serviceMock.HttpWrapper{}
 	httpWrapper.On("Get", SERIE_LINK).Return(&http.Response{
-		Body: open("inputs/episodesList.html"),
+		Body: open(t, "inputs/episodesList.html"),
 	}, nil)
 
 	httpWrapper.On("Get", EPISODE_LINK).Return(&http.Response{
-		Body: open("inputs/episode.html"),
+		Body: open(t, "inputs/episode.html"),
 	}, nil)
 
 	httpWrapper.On("Get", MIRROR_LINK).Return(&http.Response{
-		Body: open("inputs/episodeMU.html"),
+		Body: open(t, "inputs/episodeMu.html"),
 	}, nil)
 
 	headersLocation := http.Header{}
@@ -99,7 +101,8 @@ func buildHttpWrapperMock() service.HttpWrapper {
 	return &httpWrapper
 }
 
-func open(filepath string) *os.File {
-	res, _ := os.Open(filepath)
+func open(t *testing.T, filepath string) *os.File {
+	res, err := os.Open(filepath)
+	assert.NoError(t, err, "error while reading file %s", filepath)
 	return res
 }

